distribution/docs/src/main/resources/content/_managing/_running/migrate-commands.adoc

```diff
@@ -0,0 +1,34 @@
+:title: Migrate Data Command
+:type: dataManagement
+:status: published
+:summary: Data migration
+:parent: Data Migration
+:order: 00
+
+== {title}
+
+[NOTE]
+====
+Performing a data migration creates, updates, or deletes existing metacards within the system.
```

Comment:
This is a good start but it doesn't explain _why_ data migrations are sometimes necessary. In other words, if I'm an administrator, how do I know whether I need to use this command?